src/components/Modal/Modal.styled.ts

```diff
@@ -1,6 +1,6 @@
-import styled from '@emotion/styled';
-import { css } from '@emotion/react';
-import {Box} from "@mui/material";
+import { css } from '@emotion/react'
```

Comment:
Could you please use `;` as it was?
Generally, it should be done by prettier via husky but it stopped working for some reason.
I will check what is wrong 👍

Reply:
Ok i did it